### ISSUE TITLE
PersonFS blocks Gramps if there's no internet

### ISSUE THREAD

**Issue by olivierberten** [jmichault/PersonFS#21]
I tried to open Gramps without being connected to internet and PersonFS prevents me from displaying the People list....

There's of course no point in using PersonFS without internet but if that gramplet happens to be open, PersonFS should catch that situation and exit gracefully...

**Comment by jmichault** [comment]
Can you test with 1..4.4 ?

**Comment by olivierberten** [comment]
It still hangs as PersonFS is trying to update but then the list appears so I'd say it's almost fixed ;-)

```
gedcomx ne trovita aŭ < 1.0.9
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fab2c454cd0>: Failed to establish a new connection: [Errno -2] Nom ou service inconnu')': /simple/gedcomx-v1/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fab2c5161d0>: Failed to establish a new connection: [Errno -2] Nom ou service inconnu')': /simple/gedcomx-v1/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fab2c472d10>: Failed to establish a new connection: [Errno -2] Nom ou service inconnu')': /simple/gedcomx-v1/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fab2c4704d0>: Failed to establish a new connection: [Errno -2] Nom ou service inconnu')': /simple/gedcomx-v1/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fab2c4724d0>: Failed to establish a new connection: [Errno -2] Nom ou service inconnu')': /simple/gedcomx-v1/
ERROR: Could not find a version that satisfies the requirement gedcomx-v1 (from versions: none)
ERROR: No matching distribution found for gedcomx-v1
WARNING: Plugin error (from 'PersonFS'): No module named 'gedcomx'
WARNING: Error loading gramplet 'FS Gramplet': skipping content
```

**Comment by jmichault** [comment]
fix more complete in release 1.4.6.
